simple_certmanager/models.py

```diff
@@ -39,13 +39,11 @@ class SigningRequest(models.Model):
         help_text=_("Email address for the certificate"), blank=True
     )
     csr = models.TextField(
-        editable=False,
```

Comment:
there is no reason to make these fields editable and expose them through forms, we manage the values programmatically

Reply:
Oh yeah I confused editable for all types of edits (since we now allow the edition of the CSR/PK values).